build.gradle

```diff
@@ -2,6 +2,8 @@
 
 buildscript {
     ext.kotlin_version = '1.2.70'
+    ext.lifecycle_version = "2.0.0-rc01"
```

Comment:
Meanwhile lifecycle 2.0.0 and android gradle plugin 3.2.0 (with Android Studio 3.2 on stable channel) was released.
Please update these dependencies.